RestSharp/Deserializers/JsonDeserializer.cs

```diff
@@ -123,7 +122,7 @@ private IDictionary BuildDictionary(Type type, object parent)
 
                 object item;
 
-                if (valueType.IsGenericType && valueType.GetGenericTypeDefinition() == typeof(List<>))
+                if (valueType.IsGenericType() && valueType.GetGenericTypeDefinition() == typeof(List<>))
```

Comment:
Most properties of `Type` are gone in UWP and moved to `TypeInfo`. I've added extension methods with the same names so that changes are minimal. You can check those extension methods, all what they do is using properties of `Type` or `TypeInfo` depending on target.